### ISSUE TITLE
When updating titles, the inline input shrinks

### ISSUE THREAD

**Issue by ruihildt** [lambda-art-portfolio-bweu/art-portfolio-frontend#24]
When updating titles (Profile name and post title), the inline input shrinks.

Ideally we would want to have the input text and box keep the same height.

**Comment by iykekings** [comment]
Yeah, we'd have to target it with css and keep the same font-size
